map.py

```diff
@@ -184,59 +183,63 @@ def plot_solution_routes_on_map(
 
     # get colourblind palette from seaborn (10 colours) and expand if more vehicles
     palette = [
```

Comment:
(minor) this is an issue if anyone decides to use more than 10 vehicles, a bit more noticeable when the location icons are coloured as well. Maybe we could just lighten/darken each batch to be able to (somewhat) distinguish them. Might be easier to use HSB or HSL colours instead (not sure if that would work easily though).

Reply:
I don't think there's a way to change the images at all the way folium icons is set up but we could use [DivIcon](https://python-visualization.github.io/folium/latest/reference.html#folium.features.DivIcon) instead and I can just make the icons with html & maybe an svg or something which then would be easy to make it any colour we want

Reply:
I don't think it's worth spending too much time on this now. Just worth keeping in mind.